electron/launcher/src/_actions/user.actions.types.js

```diff
@@ -0,0 +1,14 @@
+export const LOGIN_START = 'LOGIN_START';
+export const LOGIN_SUCCESS = 'LOGIN_SUCCESS';
+export const LOGIN_FAILED = 'LOGIN_FAILED';
+export const LOGOUT_SUCCESS = 'LOGOUT_SUCCESS';
+export const LOGOUT_FAILED = 'LOGOUT_FAILED';
+export const REGISTER_START = 'REGISTER_START';
+export const REGISTER_SUCCESS = 'REGISTER_SUCCESS';
+export const REGISTER_FAILED = 'REGISTER_FAILED';
+export const VERIFY_EMAIL_START = 'VERIFY_EMAIL_START';
+export const VERIFY_EMAIL_SUCCESS = 'VERIFY_EMAIL_SUCCESS';
+export const VERIFY_EMAIL_FAILED = 'VERIFY_EMAIL_FAILED';
+export const RESEND_VERIFICATION_EMAIL_SUCCESS = 'RESEND_VERIFICATION_EMAIL_SUCCESS';
+export const RESEND_VERIFICATION_EMAIL_FAILED = 'RESEND_VERIFICATION_EMAIL_FAILED';
+export const RESET_ERROR = 'RESET_ERROR';
```

Comment:
probably better to make the file a less generic name - ie:
`authentication.types.js`

Reply:
Also would it also be better put in another folder? ie `_states/authentication.types.js`

Reply:
for the types thing, i think the standard is to make a `_constants` folder and our file would probably be renamed `user.types.js` or `authentication.types.js`

Reply:
Created new _constants folder and renamed to user.types.js